src/smdba/postgresqlgate.py

```diff
@@ -941,7 +941,7 @@ def do_system_check(self, *args, **params):
             changed = True
 
         # max_locks_per_transaction
-        if not conf.get('max_locks_per_transaction'):
+        if not conf.get('max_locks_per_transaction') or conf.get('max_locks_per_transaction') < '100':
```

Comment:
oh, with my version 
```python
changed = True
```

will not have the effect you wanted if it does what i think it does.